packages/dayjs/index.js

```diff
@@ -0,0 +1,7 @@
+const dayjs = require('dayjs');
+const advancedFormat = require('dayjs/plugin/advancedFormat');
+const utc = require('dayjs/plugin/utc');
+const timezone = require('dayjs/plugin/timezone');
+require('dayjs/locale/es');
```

Comment:
@Shinsina  & @zarathustra323 

Another option I see is writing some sore of util that sets this within this package to correctly include the dynamic require('dayjs/locale/es') call. Meaning a util something like this
```js
const dayjs = require('../index');

module.exports = (locale) => {
  switch (locale) {
    case 'es':
      // eslint-disable-next-line global-require
      require('dayjs/locale/es');
      dayjs.locale(locale);
      console.log('setME: ', dayjs.locale());
      break;
    default:
      // Defaults to 'en' (English) our of the box
      break;
  }
};
```

And then in the mark-web package in website-context it would have to include these `const dayjs = require('@parameter1/base-cms-dayjs');` & `const setLocale = require('@parameter1/base-cms-dayjs/utils/set-locale');`

along with the `setLocale(locale);` call below req.app.locals.markoCoreDate call.

@Shinsina & @zarathustra323  Thoughts?